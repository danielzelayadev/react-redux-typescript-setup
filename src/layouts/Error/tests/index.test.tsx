import { shallow } from 'enzyme';
import * as React from 'react';
import ErrorLayout from '..';

describe('ErrorLayout', () => {
  it('should render correctly', () => {
    expect(
      shallow(<ErrorLayout message="Whoops! Something went wrong X_X" />)
    ).toMatchSnapshot();
  });
});
