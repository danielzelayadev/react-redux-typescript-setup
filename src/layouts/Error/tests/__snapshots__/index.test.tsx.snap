// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorLayout should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ErrorLayout
    message="Whoops! Something went wrong X_X"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "background": "red",
      "children": <h1>
        Whoops! Something went wrong X_X
      </h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Whoops! Something went wrong X_X",
      },
      "ref": null,
      "rendered": "Whoops! Something went wrong X_X",
      "type": "h1",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "background": "red",
        "children": <h1>
          Whoops! Something went wrong X_X
        </h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Whoops! Something went wrong X_X",
        },
        "ref": null,
        "rendered": "Whoops! Something went wrong X_X",
        "type": "h1",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
